torchvision/csrc/io/image/cuda/encode_decode_jpeg_cuda.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include <torch/types.h>
+#include "../image_read_mode.h"
+
+#if NVJPEG_FOUND
+#include <nvjpeg.h>
+
+extern nvjpegHandle_t nvjpeg_handle;
+extern std::once_flag nvjpeg_handle_creation_flag;
+#endif
+
+namespace vision {
+namespace image {
+
+C10_EXPORT torch::Tensor decode_jpeg_cuda(
+    const torch::Tensor& data,
+    ImageReadMode mode,
+    torch::Device device);
+
+C10_EXPORT std::vector<torch::Tensor> encode_jpeg_cuda(
```

Comment:
Out of curiosity, why does this interface take in and return a std::vector<Tensor> instead of a single stacked  Tensor? I am asking because in the TorchMedia decoder, our users wanted a single stacked tensor for all the video frames and doing the torch.stack() operation on a python list of tensors was time-consuming.